web/templates/stream_settings/browse_streams_list_item.hbs

```diff
@@ -2,47 +2,50 @@
 <div class="stream-row" data-stream-id="{{stream_id}}" data-stream-name="{{name}}">
 
     {{#if subscribed}}
-        <div class="check checked sub_unsub_button tippy-zulip-tooltip" data-tooltip-template-id="unsubscribe-from-{{name}}-stream-tooltip-template">
+        <div class="check checked sub_unsub_button">
 
-            <template id="unsubscribe-from-{{name}}-stream-tooltip-template">
-                <span>
-                    {{#tr}}
-                        Unsubscribe from <z-stream></z-stream>
-                        {{#*inline "z-stream"}}{{> ../inline_decorated_channel_name stream=this}}{{/inline}}
-                    {{/tr}}
-                </span>
-            </template>
+            <div class="tippy-zulip-tooltip" data-tooltip-template-id="unsubscribe-from-{{name}}-stream-tooltip-template">
+                <template id="unsubscribe-from-{{name}}-stream-tooltip-template">
```

Comment:
We should rework this to use the stream ID rather than stream name for these IDs; can you add a commit doing that?